### PR TITLE
Plat 368 merge msg vstateunavailable

### PR DESCRIPTION
**Pull request by sexyhamster**
**- What I did**
- no VStateUnavailble, only VStateReport
- more tests on execute on empty/missing objects

**- How to verify it**
run tests

### CODE REVIEW

File: ledger-core/v2/virtual/object/object_migration_test.go [hunk 1]
```diff
@@ -59,24 +60,28 @@ func TestSMObject_MigrationFail_IfStateIsEmptyAndNoCounters(t *testing.T) {
 	smObject.SharedState.SetState(Empty)
 
 	migrationCtx := smachine.NewMigrationContextMock(mc).
+		JumpMock.
+		Inspect(func(gotJump smachine.StateFunc) {
```
Comment (bigbes) [inline ledger-core/v2/virtual/object/object_migration_test.go:64]: wrapper, pls

File: ledger-core/v2/virtual/object/object_migration_test.go [hunk 2]
```diff
@@ -59,24 +60,28 @@ func TestSMObject_MigrationFail_IfStateIsEmptyAndNoCounters(t *testing.T) {
...
+	smObject.migrate(migrationCtx)
 
 	mc.Finish()
 }
 
 func TestSMObject_MigrationJmpToStepSendVStateUnavailable_IfStateMissing(t *testing.T) {
-	var (
-		mc = minimock.NewController(t)
+	mc := minimock.NewController(t)
+	defer mc.Finish()
```
Comment (bigbes) [inline ledger-core/v2/virtual/object/object_migration_test.go:77]: get rid of defer here
